.github/workflows/ci.yaml

```diff
@@ -698,6 +698,8 @@ jobs:
             --cov-report= \
             -o console_output_style=count \
             -p no:sugar \
+            --disable-socket \
+            --allow-unix-socket \
```

Comment:
This is not so nice.
There are other integration alternative: https://github.com/miketheman/pytest-socket#usage, the best option might be to do this in the top level `conftest.py`:
```py
import pytest_socket

def pytest_runtest_setup():
    pytest_socket.disable_socket(allow_unix_socket=True)
```

Reply:
Adding the limitation to the top-level conftest.py is clearly better.

But why do we allow unix_sockets, I thought the tests should operate with mock and sort of "offline".

Reply:
unix sockets are use by asyncio, so we have to allow it.

Reply:
The limitation is now added to the top-level conftest.py instead